README.md

```diff
@@ -16,6 +16,21 @@
 >
 > Check out the [Connectors SDK](https://github.com/camunda/connectors)
 
+# Important concepts
+
+When building an inbound connector, there are several important concepts to understand:
+
+- **Activation:** When a process definition with this connector is deployed. This is a synchronous operation, so long-running tasks should be started asynchronously.
+- **[Activation condition](https://docs.camunda.io/docs/components/connectors/use-connectors/#activation-condition):** A BPMN expression that must evaluate to true for the connector to be activated.
```

Comment:
Might be a bit confusing that activation and Activation condition are so close together and yet they refer to completely different concepts